app/controllers/sitemaps_controller.rb

```diff
@@ -0,0 +1,5 @@
+class SitemapsController < ApplicationController
+  def sitemap
+    redirect_to "https://s3.amazonaws.com/#{ ENV['AWS_S3_BUCKET'] }/sitemaps/sitemap.xml.gz"
```

Comment:
Pls point to the sitemap like it was hosted in the same domain at the root. I will need to do some cloudfront magic to achieve that. (Google doesn't like it if it's not on the same domain, as it turned out.)